### PR TITLE
[V2] chore: update version and fix typo

### PR DESCRIPTION
**Pull request by hccheng72**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update version and fix a type in the README

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ schedulerExtender:
       monitor:
         enabled: true
 EOF
-helm install azuredisk-csi-driver azuredisk-csi-driver/azuredisk-csi-driver --namespace kube-system --version v2.0.0-alpha.1 --values /tmp/azure-isk-csi-driver-overrides.yaml
+helm install azuredisk-csi-driver azuredisk-csi-driver/azuredisk-csi-driver --namespace kube-system --version v2.0.0-beta.5 --values /tmp/azuredisk-csi-driver-overrides.yaml
```
Comment (landreasyan) [inline charts/README.md:285]: you can now change the version to `v2.0.0-beta.6` :)

Comment (hccheng72) [reply charts/README.md:285]: I'll change the version to beta.6 in other places as well.

**Pull request merged**